modules/machine-health-checks-baremetal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * machine_management/deploying-machine-health-checks.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-health-checks-baremetal_{context}"]
+= MachineHealthChecks on Bare Metal
+
+Machine deletion on bare metal cluster triggers reprovisioning of a bare metal host.
+Usually bare metal reprovisioning is a lengthy process, in which the cluster 
```

Comment:
I had to read this a couple of times to understand it, `during` may be more obvious what you mean here
```suggestion
Usually bare metal reprovisioning is a lengthy process, during which the cluster 
```